.github/PULL_REQUEST_TEMPLATE/regular-member-pull-request.md

```diff
@@ -7,11 +7,11 @@ about: You'd like to participate in the Cross Project Council's work as a Regula
 We are thrilled that you'd like to participate in our work in a meaningful way as a Regular Member of the Cross Project Council.
 
 Requirements and rules around becoming a Regular Member can be found in our Governance:
-https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#approving-and-onboarding-regular-members
+https://github.com/openjs-foundation/cross-project-council/blob/main/GOVERNANCE.md#approving-and-onboarding-regular-members
```

Comment:
```suggestion
https://github.com/openjs-foundation/cross-project-council/blob/HEAD/GOVERNANCE.md#approving-and-onboarding-regular-members
```
same throughout; this way they aren't hardcoded to any particular branch.

Reply:
That is a good idea. On it 👍🏽